package.json

```diff
@@ -119,6 +119,7 @@
 		"popper.js": "^1.16.1",
 		"prom-client": "^15.0.0",
 		"qrcode": "^1.4.4",
+		"qrcode.vue": "^1.7.0",
```

Comment:
Do we still need the baseline `qrcode` package? Or does the new `qrcode.vue` rely on it?

Reply:
I was wondering the same thing, it's not referenced anywhere. Do you know where we used it?

Reply:
Didn't see it used anywhere...thought maybe it was a dep of the prior library...

Reply:
Oh looks like it was installed the same time as the old package, I'll try removing it!

Reply:
Done, turns out that the `qrcode` package isn't needed by this new library.

Reply:
Cool, thank you!